.cargo/config.toml

```diff
@@ -0,0 +1,5 @@
+[build]
+rustflags = ["--cfg=web_sys_unstable_apis"]
```

Comment:
What does this mean? Is this unstable on the Rust side or JS side? Are there browser implications? Does this work on all browsers that support Wasm?

Reply:
Unstable on the rust side - it's required for use of the `web_sys::ReadableStreamDefaultController` (`ReadableStream` itself doesn't require it). It looks like the controller class isn't globally defined on browsers older than ~Feb 2020 (~7% in most jurisdictions), but calling `new ReadableStream({start, pull, cancel})` works as far back as at least Chrome 57 (Firefox 52-65 and Edge 16-18 appear to be the only versions where WebAssembly support was available before ReadableStream constructors). Eyeballing the caniuse figures, its about 0.1% of browsers with !ReadableStream && Webassembly.

I was a little leery of the potential for the Rust side method names to change, but it looks like the web-sys codegen for optional arguments (<original_method_name>_with_<optional arg>) is pretty stable.